test/test_scripts_run_task.py

```diff
@@ -391,8 +400,10 @@ def test_display_python_version_should_output_python_versions_title(
     assert ("Python version:" in capsys.readouterr().out) is True
 
 
-def test_display_python_version_should_output_python_versions(run_task_mod, capsys):
-    run_task_mod._display_python_version()
+def test_display_python_version_in_sub_process_should_output_python_versions(
```

Comment:
Improvement: I think we can seize the opportunity of testing the function more thoroughly thanks to monkeypatch, just like you did in the `test_clean_git_checkout()`. Here we can make `_check_output()` behave differently depending on the command it's given. This way, we can check whether the `except` statement works as expected.

That would be something close to (I didn't test this code): 

```python
    def _check_output(args):
        command = args[0]
        python_executable = command[0]
        if python_executable == "python2":
              raise FileNotFoundError("[Errno 2] No such file or directory: 'python2'")
        return "Dummy python 3 version"
```